packages/protocol/migrationsConfig.js

```diff
@@ -188,7 +190,8 @@ const DefaultConfig = {
       exponent: 10,
       adjustmentSpeed: 0.1,
     },
-    membershipHistoryLength: 60,
+    // MUST BE KEPT IN SYNC WITH GROUP LOCKED GOLD DURATION
+    membershipHistoryLength: 180,
```

Comment:
"_ membershipHistoryLength  The max number of entries for validator membership history."

why does this have to be as long as locked gold duration?

Reply:
I'm unclear what it means by entries

Reply:
Yeah you're totally right, oops.